src/renderer/components/Editor/PlaybackManager.tsx

```diff
@@ -85,7 +98,13 @@ const PlaybackManager = ({
       setNowPlayingWordIndex(newPlayingWordIndex);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [time, currentProject?.transcription]);
+  }, [
+    time,
+    currentProject?.transcription,
+    rangeOverride,
```

Comment:
Just want to confirm that we want to add rangeOverride to this useEffect rather than have its own useEffect. Seems to work fine, just checking is all 😄

Reply:
Yup, in this useEffect rather than its own.